### PR TITLE
Track activity screen

### PR DESCRIPTION
**Pull request by gHashTag**
#354

### CODE REVIEW

File: src/components/ui/svg-icon.tsx [hunk 1]
```diff
@@ -494,3 +494,45 @@ export const PasteIcon = ({color, ...props}: SvgProps) => (
     />
   </Svg>
 );
+
+export const UpIcon = ({
```
Comment (vivalaakam) [inline src/components/ui/svg-icon.tsx:498]: use png icons instead svg in code

File: src/screens/notification-popup.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React, {useCallback} from 'react';
+
+import {NotificationPopup} from '@app/components/notification-popup';
+import {NotificationPopupContainer} from '@app/components/ui';
+import {useTypedNavigation} from '@app/hooks';
+
+export const NotificationPopupScreen = () => {
+  const {goBack} = useTypedNavigation();
+
+  const onClickTurnOn = useCallback(() => {
```
Comment (vivalaakam) [inline src/screens/notification-popup.tsx:10]: why can't you pass goBack directly to a function without using callback

**Review by vivalaakam — COMMENTED** [review]
I propose to combine the screens `BackupNotification`, `NotificationPopup` and `TrackActivity` into a single namespace, which would begin with something that describes their essence, like `Notification...` or `Popup...` or somewhere else

**Comment by Poplavskiye** [comment]
Could you describe pls expected behavior of this feature functionality?

**Pull request merged**